adapter/sql/util.go

```diff
@@ -51,6 +63,19 @@ func Escape(config Config, field string) string {
 	return escapedField.(string)
 }
 
+func isAliasField(field string) (*aliasField, bool) {
```

Comment:
I think this function can be simplified or even put inline the escape function.

if we decide to keep it, maybe `escapeAliasField` is better name, since it's also return value, not only boolean.
also I think it's better to return 3 variable instead of aliasField struct.

a bit about returning pointer, I think we should avoid it whenever possible, because:
- non pointer value is easier to analyze by escape analysis, and it's as fast or more faster than pointer value from mini benchmark I've tried
- no need to handle null pointer